frontend/src/features/PriorNotification/slice.ts

```diff
@@ -74,6 +76,10 @@ const priorNotificationSlice = createSlice({
       state.isPriorNotificationFormDirty = action.payload
     },
 
+    setIsReportingListOpened(state, action: PayloadAction<boolean>) {
```

Comment:
Si on ferme la side window, ça reste à true ? On ne reset pas ?

Reply:
Ah j'avoue qu'on n'en a jamais parlé mais pour moi c'était mieux de garder la side window dans l'état précédent tant qu'on est train d'utiliser l'application, au cas où on la ferme par mégarde (ce qui m'arrive d'ailleurs), non ?

Reply:
Ah oui en effet why not, mais du coup ça marche bien de fermer avec un préavis + signalements et de ré-ouvrir ?